orders/tests.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now() + datetime.timedelta(days=3)
+PRICE_PER_DAY = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def create_order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                        daycare_id=DayCare.objects.get(id=1),
+                        starting_date=STARTING_DATE,
+                        finishing_date=FINISHING_DATE,
+                        price_per_day=PRICE_PER_DAY,
+                        )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self, create_order):
+        assert create_order in Order.objects.all()
+
+    def test_delete_order(self, create_order):
+        assert create_order in Order.objects.all()
+        create_order.delete()
+        assert create_order not in Order.objects.all()
+
+    def test_approve_order(self, create_order):
+        create_order.approve_order()
+
+        assert create_order.status == StatusOptions.Approved
+        assert create_order.approval_date is not None
+
+    def test_cancel_order(self, create_order):
+        create_order.cancel_order()
+
+        assert create_order.status == StatusOptions.Canceled
+        assert create_order.cancellation_date is not None
+
+    @pytest.mark.parametrize('starting_date, finishing_date, price_per_day, num_of_days, expected_result',
+                             [
+                                 (timezone.now(), timezone.now() + datetime.timedelta(days=3), 100, 3, 300),
```

Comment:
You don't need to include the time function calls in the parametrization. use them under the `create` call. the only thing that changes from test to test (in the time aspect) is the number of days. you only need to keep this parameter.

Reply:
> You don't need to include the time function calls in the parametrization. use them under the `create` call. the only thing that changes from test to test (in the time aspect) is the number of days. you only need to keep this parameter.

Removed both starting_date and finishing_date from the parametrization, and used the num_of_days for delta.